tools/cross_runner/src/mha.h

```diff
@@ -0,0 +1,490 @@
+#pragma once
+#include <vector>
+#include <random>
+
+#include "dml_base_node.h"
+#include "softmax.h"
+
+enum class MhaType
+{  
+    // qkv
+    MhaType_QKV = 1,
+};
+namespace gpu_op
+{
+    class Mha : public DirectMlBaseNode 
+    {
+    public:
+        Mha(MhaType mha_type, const DML_TENSOR_DATA_TYPE data_type, const dml::TensorPolicy& tensor_policy,
+            const TensorShape& shape_a, const TensorShape& shape_b, const TensorShape& shape_c, const TensorShape& shape_out,
```

Comment:
lets have the name more explicit, like shape_qkv,  shape_q etc.

Reply:
Currently, shape_b and shape_c is not used, I leave them here just in case they would be used for other mha type. But yes, I can move them out if it is better,